test/active_model_serializers/railtie_test_isolated.rb

```diff
@@ -0,0 +1,69 @@
+# Execute this test in isolation
```

Comment:
note that file is named `_isolated.rb` so that it can be picked up and tested separately